### ISSUE TITLE
TPTP includes are ignored

### ISSUE THREAD

**Issue by eugenk** [spechub/Hets#1535]
When trying to analyze `Problems/AGT/AGT001+1.p` from the TPTP distribution [1], Hets only parses the `fof` lines and not the `include` lines. The `include`s must also consider the catalog mechanism to switch URLs.

The Files are in the follwing gist: 

--
#### References
[1] http://www.cs.miami.edu/~tptp/TPTP/Distribution/TPTP-v6.2.0.tgz

**Comment by sternk** [comment]
The first step would probably be to allow for .p files to be read directly (instead of having to be wrapped in a specification) - I'd like to wait for #1536 and #1191 before moving forward with this.

Can you give me any information on the 'catalog mechanism'?

**Comment by eugenk** [comment]
you can analyse .p files by calling hets with
```
hets --input-type=tptp /path/to/file.p
```

**Comment by eugenk** [comment]
I don't know how the catalog mechanism is implemented. Maybe @cmaeder can give you details. The catalog mechanism basically changes names by search and replace.

If you call hets with
```
hets --url-catalog="Axioms,http://example.com/Axioms" /path/to/file
```
then every occurrence of "Axioms" in the include/import directives of the file will be replaced by "http://example.com/Axioms" for static analysis. Hets already supports this for DOL, CASL, CommonLogic and possibly OWL, but it must be supported for TPTP includes as well.

**Comment by sternk** [comment]
True, but not very well since the implementation in the SoftFOL logic which provides this) is less complete than the one in the THF logic

**Comment by sternk** [comment]
I see. Can someone maybe point me to a place in hets where this is used?

**Comment by eugenk** [comment]
Okay, this sounds to me like the referenced files are not valid tptp files... is this correct?

**Comment by cmaeder** [comment]
`Driver.Options.useCatalogURL` does the translation of input names

**Comment by sternk** [comment]
@cmaeder thanks

The file is mostly ok. Just the small issue I pointed out

**Comment by eugenk** [comment]
As far as I'm concerned with my master's thesis, this is not very urgent. I can transform those files to resolve the includes. So this issue is not very urgent from my perspecitve.
